dms/models/dms_file.py

```diff
@@ -132,6 +132,20 @@ class File(models.Model):
     # Extend inherited field(s)
     image_1920 = fields.Image(compute="_compute_image_1920", store=True, readonly=False)
 
+    model_name = fields.Char(
+        "Name of the Model",
+        compute="_compute_default_model_name",
+        readonly=True,
+        invisible=True,
+    )
+
+    # ----------------------------------------------------------
+    # Compute
+
+    def _compute_default_model_name(self):
+        for record in self:
+            record.model_name = "dms.file"
```

Comment:
Is this really necessary? I see it is used for .js but maybe you could avoid adding this field by using model.

Reply:
I removed it since it is not necessary anymore!